taxi/models.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return self.name
+
+    class Meta:
+        ordering = ('name', 'country',)
```

Comment:
The `verbose_name` and `verbose_name_plural` attributes are missing in the `Meta` class of `Manufacturer` model. It's better to add them for a more descriptive name.